kolena/_utils/scikit-learn.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021-2023 Kolena Inc.
```

Comment:
What is the purpose of this file?

Reply:
To check if sklearn is installed or not in advance, [similar to this](https://github.com/kolenaIO/kolena/blob/trunk/kolena/_utils/datatypes.py#L33). I might have misunderstood what @gordonhart had in mind.

Reply:
I think this may have to be part of your object detection PR, which would use the code here to check sklearn is installed if users want to use (import) anything in `_experimental.object_detection`.